resources/user_config.ts

```diff
@@ -53,29 +92,41 @@ class UserConfig {
     //  * b/some_file.js
     //  * root_file1.js
     //  * root_file2.js
-    return keys.sort((a, b) => {
-      const maxLen = Math.max(splitKeyMap[a].length, splitKeyMap[b].length);
+    return keys.sort((keyA, keyB) => {
+      const listA = splitKeyMap[keyA];
+      const listB = splitKeyMap[keyB];
+      if (listA === undefined || listB === undefined)
+        return 0; // Convince TypeScript these exist.
```

Comment:
Which are we convincing here?

Reply:
ts think listA and listB maybe undefined

Reply:
> Which are we convincing here?

Sorry, I'm not sure what your question is.  Are you saying that these might actually be undefined?  Can you clarify what you're trying to get at?

Reply:
I don't know what "these" represents in the comment. It's a line comment, so I feel like "these" is the zero. But, that makes no sense, so I'm assuming it's listA and listB

Reply:
Moved the comment, thanks!